monitoring/storage.go

```diff
@@ -50,17 +88,29 @@ type HighWatermarkCheckerData struct {
 	AvailableBytes uint64 `json:"available_bytes"`
 }
 
-// FailureMessage returns failure watermark check message
-func (d HighWatermarkCheckerData) FailureMessage() string {
-	return fmt.Sprintf("disk utilization on %s exceeds %v percent (%s is available out of %s), see https://gravitational.com/telekube/docs/cluster/#garbage-collection",
+// WarningMessage returns warning watermark check message
+func (d HighWatermarkCheckerData) WarningMessage() string {
+	return fmt.Sprintf("disk utilization on %s exceeds %v%% (%s is available out of %s), cluster will degrade if usage exceeds %v%%, see https://gravitational.com/gravity/docs/cluster/#garbage-collection",
```

Comment:
Sort of in relation to above, it seems slightly inconsistent to report that we exceed a threshold of 80%, but then report usage in bytes and not %.

Reply:
I think it's nice to be able to see how much disk is available in bytes, but yeah inconsistent with the percentage. We can include both, though the message is already quite long.